core/cc/log.h

```diff
@@ -132,6 +98,14 @@ class Logger {
 
 }  // namespace core
 
+#if TARGET_OS == GAPID_OS_ANDROID
+#include <android/log.h>
+#else  // TARGET_OS == GAPID_OS_ANDROID
+#include <stdio.h>
+#endif  // TARGET_OS == GAPID_OS_ANDROID
```

Comment:
Can these includes now be moved to the .cpp?

Reply:
Not unless we move them to (potentially) every c++ that uses log.h. This is quite a bit more than just Log.cpp

Reply:
Done!